### ISSUE TITLE
ng-bootstrap button group active state

### ISSUE THREAD

**Issue by gfellerph** [swisspost/common-web-frontend#111]
![image](https://user-images.githubusercontent.com/1659006/170215909-9bbf888e-2cd9-4a57-af0c-46e84e3cfabe.png)

Button groups don't indicate the currently active button.

**Comment by gfellerph** [comment]
Cannot reproduce with examples in the demo app.
